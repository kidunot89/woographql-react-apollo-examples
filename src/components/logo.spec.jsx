import React from 'react';
import '@testing-library/react/dont-cleanup-after-each';
import { render, cleanup } from '@testing-library/react';

import Logo from './logo';

describe('Logo component', () => {
  afterAll(() => cleanup());
  const { baseElement } = render(
    <Logo />,
  );

  it('should render successfully', () => {
    expect(baseElement).toMatchSnapshot();
  });
});
