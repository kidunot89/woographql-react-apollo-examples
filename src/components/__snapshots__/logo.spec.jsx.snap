// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo component should render successfully 1`] = `
<body>
  <div>
    <div
      class="inline-flex items-center"
    >
      <div
        class="h-14 w-14 max-h-[90%]"
      >
        <svg
          class="w-full h-full"
          height="100"
          viewBox="0 0 100 100"
          width="100"
        >
          <circle
            class="stroke-shark fill-blue-bayoux stroke-[3]"
            cx="50"
            cy="50"
            r="40"
          />
        </svg>
      </div>
      <h1
        class="my-0 ml-3 hidden font-serif text-3xl font-bold text-shark lg:block"
      >
        ¯\\_(ツ)_/¯
      </h1>
    </div>
  </div>
</body>
`;
